HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -188,6 +189,13 @@ def self.apply_header(hpxml, epw_file)
       hpxml.header.use_max_load_for_heat_pumps = true
       hpxml.header.use_max_load_for_heat_pumps_isdefaulted = true
     end
+
+    if (not epw_file.nil?) && hpxml.header.state_code.nil?
+      if Constants.StateCodes.include? epw_file.stateProvinceRegion
```

Comment:
Why restrict the defaults here? This doesn't seem necessary.

Reply:
`base-location-capetown-zaf.xml` returns a `-`, for example.

Reply:
Maybe we restrict the defaults to being at least a single alpha character, or something?

Reply:
I don't see why we can't just use whatever is in the epw file. What is the harm?

Reply:
Seems weird to default to a "-" character for a "state code"...

Reply:
I put in a check that `epw_file.stateProvinceRegion` is two alpha characters.